WEBPACK_UPDATE.md

```diff
@@ -0,0 +1,59 @@
+# Upgrading from angular-cli@1.0.0-beta.10 to the angular-cli@1.0.0-beta.11-webpack 
+
+To update `angular-cli` to the webpack preview, you must update both the global package, project's local package and your project files.
```

Comment:
Technically the global package shouldn't care about it. Only the local package needs to be installed.